src/schemas/Program/index.js

```diff
@@ -96,6 +96,22 @@ const typeDefs = gql`
 		countries: [String]!
 	}
 
+	type DataCenter {
+		id: ID
+		shortName: String!
+		name: String
+		organization: String
+		email: String
+		uiUrl: String
+		gatewayUrl: String
+		analysisSongCode: String
+		analysisSongUrl: String
+		analysisScoreUrl: String
+		submissionSongCode: String
+		submissionSongUrl: String
+		submissionScoreUrl: String
+	}
+
```

Comment:
gql type. made only the shortName restrictive, following what is used for Program type.